dv/uvm/core_ibex/sim.py

```diff
@@ -413,6 +413,43 @@ def compare(test_list, iss, output_dir):
     return fails == 0
 
 
+#TODO(udinator) - support IUS, DSim, and Riviera
+def gen_cov(base_dir, simulator, lsf_cmd):
+    """Generate a merged coverage directory, and open up the appropriate GUI
+
+    Args:
+        base_dir:   the base simulation output directory (default: out/)
+        simulator:  the chosen RTL simulator
+        lsf_cmd:    command to run on LSF
+
+    """
+    # Compile a list of all output seed-###/rtl_sim/test.vdb directories
+    dir_list = []
+    for entry in os.scandir(base_dir):
+        vdb_path = "%s/%s/rtl_sim/test.vdb" % (base_dir, entry.name)
+        if 'seed' in entry.name:
+            logging.info("Searching %s/%s for coverage database" %
+                          (base_dir, entry.name))
+            if os.path.exists(vdb_path):
+                dir_list.append(vdb_path)
+    if dir_list == []:
+        logging.info("No coverage data available, exiting...")
+        sys.exit(RET_SUCCESS)
+
+    if simulator == 'vcs':
+        cov_cmd = "urg -full64 -format both -dbname test.vdb " \
+                  "-report %s/rtl_sim/urgReport -dir " % base_dir
+        for cov_dir in dir_list:
+            cov_cmd += "%s " % cov_dir
```

Comment:
Minor nit: I think you've got a trailing space here. Probably drop the trailing space after `-dir` and then change this to `" %s"`.

Reply:
Good catch, I've updated the code accordingly